mirgecom/inviscid.py

```diff
@@ -128,32 +128,183 @@ def inviscid_facial_flux(discr, state_tpair, local=False):
         the face normals as required by the divergence operator for which they
         are being computed.
     """
```

Comment:
Documented arguments need updating (or maybe removal, to match `inviscid_flux_hll`); they don't match the function prototype.

Reply:
Fixed this up.

Reply:
Heh, sorry -- I meant to comment on `inviscid_flux_rusanov`, but I guess I scrolled down too far. What I was getting at here is that `inviscid_flux_rusanov` has documentation for arguments that are no longer present (`discr`, `local`, etc.). Presumably they should be made to look more like the docs for `inviscid_flux_hll`.